### PR TITLE
Cleanup tests properly

### PR DESCRIPTION
**Pull request by chadhietala**
Premature commit on #5666

### CODE REVIEW

**Comment by stefanpenner** [comment]
> Premature commit on #5666

happens to the best of us

**Comment by stefanpenner** [comment]
:+1:

File: tests/unit/models/file-info-test.js [hunk 1]
```diff
@@ -110,7 +111,7 @@ describe('Unit - FileInfo', function() {
   });
 
   it('renders a menu with an overwrite option', function() {
-    var originalPrompt = ui.prompt;
+    originalPrompt = ui.prompt;
```
Comment (Turbo87) [inline tests/unit/models/file-info-test.js]: shouldn't this be in the `beforeEach` block too?

**Comment by Turbo87** [comment]
@chadhietala 

```
  1) Blueprint basic blueprint installation "before each" hook for "installs basic files":
     TypeError: Cannot read property 'prompt' of undefined
      at Context.<anonymous> (tests/unit/models/blueprint-test.js:475:26)
  2) Blueprint basic blueprint installation "after each" hook for "installs basic files":
     TypeError: Cannot set property 'prompt' of undefined
      at Context.<anonymous> (tests/unit/models/blueprint-test.js:492:17)
```

File: tests/unit/models/blueprint-test.js [hunk 2]
```diff
@@ -468,8 +468,13 @@ help in detail');
     var project;
     var options;
     var tmpdir;
+    var originalPrompt;
+    var prompt;
 
     beforeEach(function() {
+      originalPrompt = ui.prompt;
+      prompt = ui.prompt = td.function();
```
Comment (Turbo87) [inline tests/unit/models/blueprint-test.js]: `ui` hasn't been set at this point. see line 476 below.

**Comment by chadhietala** [comment]
@homu r+

**Pull request merged**